testing/regress/hpcc/regression/suite.py

```diff
@@ -126,8 +126,17 @@ def addFileToSuite(self, eclfile):
             versions = eclfile.getVersions()
             versionId = 1
             for version in versions:
-                files.append({'basename':basename, 'version':version,  'id':versionId })
-                versionId += 1
+                if 'no'+self.name in version:
+                    # Exclude it from this target
+                    pass
+                else:
+                    # Remove exclusion key(s) from version string
+                    config = getConfig();
```

Comment:
(Above I mistyped the name, so the correct sentence is:” Or if you want to //nohthor tag excludes the file from all clusters which name starts with hthor then I think, it is a different issue.)
It wasn't problem in the last 1 - 1.5 year and we have couple of hundred official test results with this approach. Maybe the Clusters name (of the array in the config) is not accurate, we can change it easily. 
It is possible the terms (target, platform, cluster , etc.) are a little bit foggy and/or only in mind defined and/or mismatched/misused. We can clarify them easily too. 
BTW Actually the 

//version ... ,no<hthor|thor|roxie>

works in same way as the 

//no<hthor|thor|roxie>

tag does.  Therefore any or more no<hthor|thor|roxie> is defined in the //version tag then the

 ./ecl-test setup|run|query -t <hthor|thor|roxie> …

command doesn’t execute that version with 

ecl run –t <hthor|thor|roxie> …

command on the defined target.

If this approach is wrong then please raise a JIRA for this and specify how it should work.